packages/engine/paima-sm/src/cde-erc721-transfer.ts

```diff
@@ -21,7 +29,15 @@ export default async function processErc721Datum(
     );
     const newOwnerData = { cde_id: cdeId, token_id: tokenId, nft_owner: toAddr };
     if (ownerRow.length > 0) {
-      updateList.push([cdeErc721UpdateOwner, newOwnerData]);
+      if (isBurn) {
+        updateList.push([
```

Comment:
Probably good to leave a comment explaining that we're doing this so we can track the address that owned the NFT before it was sent to the burn address